packages/core/src/filters/Filter.ts

```diff
@@ -212,11 +223,13 @@ export class Filter extends Shader
      *        There are some useful properties in the currentState :
      *        target, filters, sourceFrame, destinationFrame, renderTarget, resolution
      */
-    apply(filterManager, input, output, clearMode, currentState)
+    apply(filterManager: FilterSystem, input: RenderTexture, output: RenderTexture, clearMode: CLEAR_MODES,
+        /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+        _currentState?: FilterState): void
     {
         // do as you please!
 
-        filterManager.applyFilter(this, input, output, clearMode, currentState);
+        filterManager.applyFilter(this, input, output, clearMode);
```

Comment:
Why was this code changed?

Reply:
because that parameter is absent in filterManager.